driver/others/dynamic_power.c

```diff
@@ -202,7 +202,7 @@ static gotoblas_t *get_coretype(void) {
 		return &gotoblas_POWER10;
 #endif
 	/* Fall back to the POWER9 implementation if the toolchain is too old or the MMA feature is not set */
-#if (!defined __GNUC__) || ( __GNUC__ >= 6)
+#if (!defined __GNUC__) || ( __GNUC__ >= 11)
```

Comment:
@martin-frbg Since its just a fall back to POWER9, can we just move lines if (__builtin_cpu_is("power10"))
                return &gotoblas_POWER9;  inside ifdef HAVE_P10_SUPPORT after line 202.

Reply:
HAVE_P10_SUPPORT depends on having a very recent ld version in addition to new gcc, so a bit more than required for a fallback to POWER9

Reply:
@martin-frbg Then should we also consider gcc 10.2 here?
#if (!defined __GNUC__) || ( __GNUC__ >= 11)  || (__GNUC__ == 10 && __GNUC_MINOR__ >= 2)

Reply:
I do not think that 10.2 (released in july) already contains a patch that was committed to the gcc mainline at the end of june ?

Reply:
Link for gcc backport: https://gcc.gnu.org/git/?p=gcc.git;a=commit;h=41b504e91e5cd0fdce36b7306ce42dd592c401f2